seatunnel-connectors-v2/connector-qdrant/src/main/java/org/apache/seatunnel/connectors/seatunnel/qdrant/source/QdrantSource.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel.connectors.seatunnel.qdrant.source;
+
+import org.apache.seatunnel.api.configuration.ReadonlyConfig;
+import org.apache.seatunnel.api.source.Boundedness;
+import org.apache.seatunnel.api.table.catalog.CatalogTable;
+import org.apache.seatunnel.api.table.catalog.CatalogTableUtil;
+import org.apache.seatunnel.api.table.type.SeaTunnelRow;
+import org.apache.seatunnel.connectors.seatunnel.common.source.AbstractSingleSplitReader;
+import org.apache.seatunnel.connectors.seatunnel.common.source.AbstractSingleSplitSource;
+import org.apache.seatunnel.connectors.seatunnel.common.source.SingleSplitReaderContext;
+import org.apache.seatunnel.connectors.seatunnel.qdrant.config.QdrantConfig;
+import org.apache.seatunnel.connectors.seatunnel.qdrant.config.QdrantParameters;
+
+import java.util.Collections;
+import java.util.List;
+
+public class QdrantSource extends AbstractSingleSplitSource<SeaTunnelRow> {
+    private final QdrantParameters qdrantParameters;
+    private final CatalogTable catalogTable;
+
+    @Override
+    public String getPluginName() {
+        return QdrantConfig.CONNECTOR_IDENTITY;
+    }
+
+    public QdrantSource(ReadonlyConfig readonlyConfig) {
+        this.qdrantParameters = new QdrantParameters(readonlyConfig);
+        this.catalogTable = CatalogTableUtil.buildWithConfig(readonlyConfig);
```

Comment:
Does `Qdrant` can support get catalogtable from `Catalog`? Instead of use schema options.

Reply:
As in a schema? No.
The payload for each entry can be arbitrary JSON.

Reply:
When a collection defined, can we get it schema by `payload_schema` fields with https://qdrant.tech/documentation/concepts/collections/#collection-info api?

Reply:
`payload_schema` only contains fields of the JSON payload for which an index is created.
https://qdrant.tech/documentation/concepts/indexing/#payload-index

Reply:
I got it. Thanks for your share doc and explain!